types/tabulator-tables/index.d.ts

```diff
@@ -1742,6 +1761,9 @@ declare namespace Tabulator {
 
     interface TextAreaParams extends SharedEditorParams {
         verticalNavigation?: 'editor' | 'table' | 'hybrid' | undefined;
+
+        /** Allow submission of the value of the editor when the shift and enter keys are pressed togeather. */
```

Comment:
Typo: togeather

Reply:
Good catch. You would think I would have learned by now to spell check the release notes. 🤦